src/labels/CustomLabels.labels

```diff
@@ -12731,4 +12731,107 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
         <shortDescription>stgValidationHHAccountHHRules</shortDescription>
         <value>The Account Model is set to &apos;Household Account&apos;, which requires Household Rules to be set to &apos;No Contacts&apos;.   When using Household Accounts, there is no need to have an additional Household Object.</value>
     </labels>
+
```

Comment:
There should be no blank lines between labels, plus these new labels must be placed within the file in the right spot alphabetically.